healthcheck.js

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env node
+'use strict'
+require('dotenv').config()
+
+const fetch = require('node-fetch')
+
+const signale = require('./src/utils/signale')
+const checkMongoDB = require('./src/utils/connect')
+
+const port = process.env.ACKEE_PORT || process.env.PORT || 3000
+const dbUrl = process.env.ACKEE_MONGODB || process.env.MONGODB_URI
+const serverUrl = `http://localhost:${ port }`
+
+if (dbUrl == null) {
+	signale.fatal('MongoDB connection URI missing in environment')
+	process.exit(1)
+}
+
+const checkHTTP = async (url) => {
+	return await fetch(url)
+		.then((res) => {
+			if (res.status < 200 || res.status >= 300) {
```

Comment:
Small tip from my side: res.ok is a boolean and does exactly this check :)

Reply:
I was unaware of `res.ok`, feel free to replace it.